src/patternfly/components/TextInputGroup/text-input-group.scss

```diff
@@ -22,30 +22,31 @@
   --#{$text-input-group}__text--BorderEndStartRadius: var(--#{$text-input-group}__text--BorderRadius--base);
 
   // Label group
-  --#{$text-input-group}--c-label-group__main--PaddingBlockStart: var(--pf-t--global--spacer--xs);
-  --#{$text-input-group}--c-label-group__main--PaddingInlineEnd: var(--pf-t--global--spacer--xs);
-  --#{$text-input-group}--c-label-group__main--PaddingBlockEnd: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}--c-label-group__main--PaddingBlockStart: calc(var(--pf-t--global--spacer--control--vertical--default) / 2);
+  --#{$text-input-group}--c-label-group__main--PaddingInlineEnd: calc(var(--pf-t--global--spacer--control--horizontal--plain) / 2);
+  --#{$text-input-group}--c-label-group__main--PaddingBlockEnd: calc(var(--pf-t--global--spacer--control--vertical--default) / 2);
 
   // Text input
-  --#{$text-input-group}__text-input--PaddingBlockStart: var(--pf-t--global--spacer--sm);
-  --#{$text-input-group}__text-input--PaddingInlineEnd: var(--pf-t--global--spacer--sm);
-  --#{$text-input-group}__text-input--PaddingBlockEnd: var(--pf-t--global--spacer--sm);
-  --#{$text-input-group}__text-input--PaddingInlineStart: var(--pf-t--global--spacer--sm);
+  --#{$text-input-group}__text-input--PaddingBlockStart: var(--pf-t--global--spacer--control--vertical--default);
+  --#{$text-input-group}__text-input--PaddingInlineEnd: var(--pf-t--global--spacer--control--horizontal--plain);
+  --#{$text-input-group}__text-input--PaddingBlockEnd: var(--pf-t--global--spacer--control--vertical--default);
+  --#{$text-input-group}__text-input--PaddingInlineStart: var(--pf-t--global--spacer--control--horizontal--plain);
   --#{$text-input-group}__text-input--MinWidth: 12ch;
   --#{$text-input-group}__text-input--m-hint--Color: var(--pf-t--global--text--color--placeholder);
   --#{$text-input-group}__text-input--placeholder--Color: var(--pf-t--global--text--color--placeholder);
   --#{$text-input-group}__text-input--BackgroundColor: transparent;
   --#{$text-input-group}__text-input--OutlineOffset: -6px;
 
   // Icon
+  --#{$text-input-group}__icon--FontSize: var(--pf-t--global--icon--size--font--body--default);
   --#{$text-input-group}__icon--InsetInlineStart: var(--pf-t--global--spacer--sm);
   --#{$text-input-group}__icon--Color: var(--pf-t--global--icon--color--regular);
   --#{$text-input-group}__icon--TranslateY: -50%;
 
   // Utilities
-  --#{$text-input-group}__utilities--child--MarginInlineStart: var(--pf-t--global--spacer--xs);
-  --#{$text-input-group}__utilities--c-button--PaddingInlineEnd: var(--pf-t--global--spacer--xs);
-  --#{$text-input-group}__utilities--c-button--PaddingInlineStart: var(--pf-t--global--spacer--xs);
+  --#{$text-input-group}__utilities--child--MarginInlineStart: calc(var(--pf-t--global--spacer--control--horizontal--plain) / 2);
```

Comment:
I don't know if we have a spacer for this. `--pf-t--global--spacer--gap--action-to-action--plain` mayyy work, since I think the intention is we'll have plain actions here, but we have other things, too. IMO that might be alright, but we could err on the safe side and just continue to use `--pf-t--global--spacer--xs`. @lboehling wdyt? This is the gap used for the buttons and badge inside of the text input group on the right in this example https://patternfly-react-v6.surge.sh/components/search-input#match-with-navigable-options. Technically, I think you can put any kind of thing there you want.

<img width="851" alt="Screenshot 2024-06-13 at 8 40 28 PM" src="https://github.com/patternfly/patternfly/assets/35148959/339230da-3a3a-406e-afb6-55f258ec9e01">

Reply:
Reverted to t-shirt size for now